Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -473,17 +496,27 @@ bool PPCSymbolDB::SaveSymbolMap(const std::string& filename) const
   for (const auto& symbol : function_symbols)
   {
     // Write symbol address, size, virtual address, alignment, name
-    f.WriteString(fmt::format("{0:08x} {1:08x} {2:08x} {3} {4}\n", symbol->address, symbol->size,
-                              symbol->address, 0, symbol->name));
+    std::string line = fmt::format("  {0:08x} {1:06x} {2:08x} {3} {4}", symbol->address,
+                                   symbol->size, symbol->address, 0, symbol->name);
+    // Also write the object name if it exists
+    if (!symbol->object_name.empty())
+      line += fmt::format(" \t{0}", symbol->object_name);
+    line += "\n";
+    f.WriteString(line);
   }
 
   // Write .data section
   f.WriteString("\n.data section layout\n");
   for (const auto& symbol : data_symbols)
   {
     // Write symbol address, size, virtual address, alignment, name
-    f.WriteString(fmt::format("{0:08x} {1:08x} {2:08x} {3} {4}\n", symbol->address, symbol->size,
-                              symbol->address, 0, symbol->name));
+    std::string line = fmt::format("  {0:08x} {1:06x} {2:08x} {3} {4}", symbol->address,
+                                   symbol->size, symbol->address, 0, symbol->name);
```

Comment:
As a note: the format for the `size` was changed from `{:08x}` to `{:06x}`. I don't think this change is harmful.